content/courses/intro-to-embedded-sys-prog/chapters/low_level_programming.rst

```diff
@@ -245,60 +258,53 @@ though.
 Note that :ada:`Address` is the type of the result of the query attribute
 :ada:`Address`.
 
-We mentioned potentially needing to swap bytes in networking
-communications software, due to the differences in the "endianness" of
-the machines communicating. That characteristic is specified in :ada:`System`
-as follows:
+We mentioned potentially needing to swap bytes in networking 
+communications software, due to the differences in the "endianness" of 
+the machines communicating. That characteristic can be determined via a 
+constant declared in package :ada:`System` as follows: 
 
 .. code-block:: ada
 
    type Bit_Order is (High_Order_First, Low_Order_First);
    Default_Bit_Order : constant Bit_Order := implementation-defined;
 
-:ada:`High_Order_First` corresponds to "big-endian" and
-:ada:`Low_Order_First` to little-endian. Here is a real-world example of
-use, in which we retrieve arbitrarily-typed values from a given buffer
-starting at a given index:
+:ada:`High_Order_First` corresponds to "Big Endian" and 
+:ada:`Low_Order_First` to "Little Endian." On a Big Endian machine, bit 
+0 is the most significant bit. On a Little Endian machine, bit 0 is the 
+least significant bit. 
+
+Strictly speaking, this constant gives us the default order for bits 
```

Comment:
We've actually removed references to big/little endianess from the previous book (see https://github.com/AdaCore/learn/pull/447/files#r532750100 on line 919.)

Reply:
I assume it's ok here though

Reply:
I left them off of the discussion on record rep clauses, as being beyond the scope of the course. Butt this is just how to determine the endina-ness, so IMO we should have it.